### PR TITLE
Timer

### PR DESCRIPTION
**Pull request by ESeNonFossiIo**
```
=============================================================================================================================================

                                                    TimeMonitor results over 2 processors

Timer Name                                Local time (num calls)    MinOverProcs     MeanOverProcs    MaxOverProcs     MeanOverCallCounts
---------------------------------------------------------------------------------------------------------------------------------------------
Assemble matrices                         5.529 (14)                5.529 (14)       5.529 (14)       5.529 (14)       0.3949 (14)
Epetra_CrsMatrix::Multiply(TransA,X,Y)    0.3568 (2976)             0.3568 (2976)    0.3781 (2976)    0.3993 (2976)    0.000127 (2976)
Postprocessing                            1.47 (22)                 1.422 (22)       1.446 (22)       1.47 (22)        0.06574 (22)
Residual                                  4.004 (63)                4.004 (63)       4.028 (63)       4.053 (63)       0.06394 (63)
Setup Jacobian                            8.655 (14)                8.655 (14)       8.669 (14)       8.683 (14)       0.6192 (14)
Setup dof systems                         0.0371 (1)                0.0371 (1)       0.0371 (1)       0.0371 (1)       0.0371 (1)
Solve system                              3.269 (28)                3.231 (28)       3.25 (28)        3.269 (28)       0.1161 (28)
=============================================================================================================================================
```

### CODE REVIEW

File: source/pidomus.cc [hunk 1]
```diff
@@ -739,10 +722,11 @@ void piDoMUS<dim, spacedim, LAC>::run ()
...
   eh.output_table(pcout);
 
-  // std::ofstream f("errors.txt");
-  computing_timer.print_summary();
-  timer_outfile.close();
-  // f.close();
+  if(output_timer)
+  {
+    pcout << std::endl;
+    Teuchos::TimeMonitor::summarize(std::cout,true);
```
Comment (luca-heltai) [inline source/pidomus.cc:728]: why std::cout instead of pcout? does it take care of mpi output by itself?

File: tests/parameters/compressible_neo_hookean_01.prm [hunk 2]
```diff
@@ -137,6 +137,6 @@ subsection piDoMUS<3, 3, LATrilinos>
   set Overwrite Newton's iterations                  = true
   set Print some useful informations about processes = true
   set Time stepper                                   = ida
-  set Timer output file                              = timer.txt
```
Comment (luca-heltai) [inline tests/parameters/compressible_neo_hookean_01.prm:140]: Why did you remove the output to a file? Can't we have both?

File: source/pidomus.cc (hunk 1 above)
Comment (ESeNonFossiIo) [inline source/pidomus.cc:728]: yup. ConditionalStream is not accepted by Teuchos

Comment (ESeNonFossiIo) [inline source/pidomus.cc:728]: BTW the parallel output is ok. It takes care of mpi output.
